### ISSUE TITLE
Microsoft UWP-Strategie and regarding .NET and Core .NET

### ISSUE THREAD

**Issue by Dravion** [hmailserver/hmailserver-net#2]
Hi Martin,
I dont know if you stubled allready about this but it seems MS wants to abandoned .NET and Native Win32 more and more. Win32 and Classic .NET Developers will be "motivated" to migrate their 
Projects to the UWP-Platform. An AppConverter (for existing, binary programs) is ready to use https://developer.microsoft.com/en-us/windows/bridges/desktop to convert existing Apps but this
will not be the best way to deal with the Situation.

However:
I noticed in VS2017 a lot of older Win32 API's are now declared as deprecated, even then old SYSTEMINFORMATION structure. Its now marked as Error and you need to bypass it via Linke /wd switch.

In short:
Everything changes, even the installer and MS is trying to force us into its new and awesome Windows Store. There is a way called "Site of Business Line" Apps which can avoid the forced Windows Store bindings but it requires a x509 CA validated certificate in the Windows Trusttore first to avoid installer problems.

This will not be "real" problem for some time but someday it will have an impact on any classic existing
.NET and Win32 code.
